### PR TITLE
READY: Refactoring the render_GET method

### PR DESCRIPTION
**Pull request by vandenheuvel**
I split up the `render_GET` method of the multichain network endpoint.

### CODE REVIEW

File: Tribler/Core/Modules/restapi/multichain_endpoint.py [hunk 1]
```diff
@@ -294,26 +289,63 @@ def render_GET(self, request):
...
+        Note that once a dummy database is set, this dummy database is used until the software is restarted.
+        :param dataset: the dummy dataset type
+        :return:
+        """
+        if isinstance(dataset, basestring):
+            if dataset == "static" or dataset == "random":
+                mc_community = self.get_multi_chain_community()
+                if dataset == "static":
+                    mc_community.persistence.use_dummy_data(use_random=False)
+                elif dataset == "random":
```
Comment (yinghaodai) [inline Tribler/Core/Modules/restapi/multichain_endpoint.py]: `else` will suffice

File: Tribler/Core/Modules/restapi/multichain_endpoint.py [hunk 2]
```diff
@@ -294,26 +289,63 @@ def render_GET(self, request):
...
+        if focus_node == "self":
+            mc_community = self.get_multi_chain_community()
+            if mc_community.persistence.dummy_setup:
+                focus_node = "00"
+            else:
+                focus_node = hexlify(mc_community.my_member.public_key)
+        return focus_node
+
+    @staticmethod
+    def get_neighbor_level(arguments):
```
Comment (ghabbenjansen) [inline Tribler/Core/Modules/restapi/multichain_endpoint.py]: Why is this a static method and why do you give it the full list of arguments where the other methods only get the specific argument? (get_focus_node gets request.args["focus_node"], this one only gets request.args)

File: Tribler/Core/Modules/restapi/multichain_endpoint.py [hunk 3]
```diff
@@ -294,26 +289,63 @@ def render_GET(self, request):
...
+                    mc_community.persistence.use_dummy_data(use_random=True)
+
+    def get_focus_node(self, focus_node):
+        """
+        Set the focus node.
+
+        :return: the focus node value
+        """
+        if focus_node == "self":
+            mc_community = self.get_multi_chain_community()
```
Comment (ghabbenjansen) [inline Tribler/Core/Modules/restapi/multichain_endpoint.py]: Maybe pass the community as a parameter

File: Tribler/Core/Modules/restapi/multichain_endpoint.py [hunk 4]
```diff
@@ -294,26 +289,63 @@ def render_GET(self, request):
...
+        """
+        Set a dummy database for the rest of this run.
+
+        Note that once a dummy database is set, this dummy database is used until the software is restarted.
+        :param dataset: the dummy dataset type
+        :return:
+        """
+        if isinstance(dataset, basestring):
+            if dataset == "static" or dataset == "random":
+                mc_community = self.get_multi_chain_community()
```
Comment (ghabbenjansen) [inline Tribler/Core/Modules/restapi/multichain_endpoint.py]: Maybe add the community as a parameter

File: Tribler/Core/Modules/restapi/multichain_endpoint.py (hunk 1 above)
Comment (ghabbenjansen) [inline Tribler/Core/Modules/restapi/multichain_endpoint.py]: It won't, any argument other than "static" and "random" should result in no change in database

**Comment by MaxVanDeursen** [comment]
Make sure to make a test for the missed coverage

**Comment by rkeulemans** [comment]
retest this please

**Pull request merged**